Form-Controls/index.html

```diff
@@ -1,28 +1,103 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <title>My form exercise</title>
+      <meta name="description" content="t-shirt">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+      <header>
+        <h1>Product Form</h1>
+        
+      </header>
+      <main>
+        <div class="container">
+      <form>
+          <!-- write your html here-->
+        <!-- try writing out the requirements first-->
+
+    <h2>T-Shirt Order Form</h2>
+    <form id="tshirtOrderForm">
+        <label for="name">Your Name:</label>
+        <input type="text" id="name" name="name" required pattern="[A-Za-z\s]+" title="Please enter a valid name (letters and spaces only).">
+
+        <label for="email">Email :</label>
+        <input type="email" id="email" name="email" required>
+
+        <label for="color">T-Shirt Color:</label>
+        <select id="color" name="color" required>
+            <option value="" disabled selected>Select a color</option>
+            <option value="black">Black</option>
+            <option value="blue">Blue</option>
+            <option value="green">Green</option>
+            <option value="white">White</option>
+            <option value="pink">Pink</option>
+        </select>
+
+        <label for="qtt">Quantity:</label>
+        <input type="number" id="qtt" name="quantity" required><br>
+        
+        <label for="size">T-Shirt Size:</label>
+        <select id="size" name="size" required>
+            <option value="" disabled selected>Select a size</option>
+            <option value="XS">XS</option>
+            <option value="S">S</option>
+            <option value="M">M</option>
+            <option value="L">L</option>
+            <option value="XL">XL</option>
+            <option value="XXL">XXL</option>
+            <option value="XXXL">XXXL</option>
+    </select>
+
+        <div class="delivery-container">
+          <h2>Delivery address</h2>
+          <form action="#" method="POST">
```

Comment:
Oh, we've started a new form! 

Forms cannot be nested. Explore a bit as to why this might be https://webaim.org/techniques/forms/controls

Reply:
Used w3schools guidance. New form edited and added corrections.